extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -156,6 +156,9 @@ export class AmpSlideScroll extends BaseSlides {
    */
   touchMoveHandler_() {
     this.clearAutoplay();
+    if (!this.hasNativeSnapPoints_) {
+      return;
+    }
```

Comment:
I feel `cancelTouchEvent_()` should be called only once. But `touchMoveHandler_` is called everytime we receive `touchmove` event